packages/netlify-cms-core/src/lib/__tests__/registry.spec.js

```diff
@@ -45,7 +45,7 @@ describe('registry', () => {
   });
 
   describe('eventHandlers', () => {
-    const events = ['prePublish', 'postPublish', 'preUnpublish', 'postUnpublish'];
+    const events = ['prePublish', 'postPublish', 'preUnpublish', 'postUnpublish', 'prePush'];
```

Comment:
Should we rename it to `preSave` to match the UI term and be less "technical"? Also should we add a `postSave` for consistency?

Reply:
Yes, we should, initially thought the `postPublish` hook should suffice, but that hook will only run for non-workflow.